Sources/StytchUI/StytchB2BUIClient/ViewControllers/B2BAuthHomeViewController/B2BAuthHomeViewController.swift

```diff
@@ -61,45 +61,86 @@ final class B2BAuthHomeViewController: BaseViewController<B2BAuthHomeState, B2BA
+                    state: .init(configuration: viewModel.state.configuration),
+                    delegate: self
+                )
+                addChild(passwordsHomeViewController)
+                stackView.addArrangedSubview(passwordsHomeViewController.view)
+                constraints.append(passwordsHomeViewController.view.widthAnchor.constraint(equalTo: stackView.widthAnchor))
             case .oAuthButtons:
-                let oauthController = B2BOAuthViewController(state: .init(configuration: viewModel.state.configuration), delegate: self)
+                let oauthController = B2BOAuthViewController(
+                    state: .init(configuration: viewModel.state.configuration),
+                    delegate: self
+                )
                 addChild(oauthController)
                 stackView.addArrangedSubview(oauthController.view)
                 constraints.append(oauthController.view.widthAnchor.constraint(equalTo: stackView.widthAnchor))
             case .ssoButtons:
-                break
+                let ssoViewController = B2BSSOViewController(
+                    state: .init(configuration: viewModel.state.configuration),
+                    delegate: self
+                )
+                addChild(ssoViewController)
+                stackView.addArrangedSubview(ssoViewController.view)
+                constraints.append(ssoViewController.view.widthAnchor.constraint(equalTo: stackView.widthAnchor))
             case .divider:
                 stackView.addArrangedSubview(separatorView)
                 constraints.append(separatorView.widthAnchor.constraint(equalTo: stackView.widthAnchor))
             }
         }
 
-        if StytchClient.disableSdkWatermark == false {
-            stackView.addArrangedSubview(poweredByStytch)
-        }
-
-        stackView.addArrangedSubview(SpacerView())
-
         NSLayoutConstraint.activate(constraints)
-
-        Task {
-            try await viewModel.logRenderScreen()
-        }
     }
 }
 
 extension B2BAuthHomeViewController: B2BOAuthViewControllerDelegate {
     func oauthDidAuthenticatie() {
-        startMFAFlowIfNeeded(configuration: viewModel.state.configuration)
+        Task { @MainActor in
+            startMFAFlowIfNeeded(configuration: viewModel.state.configuration)
+        }
     }
 }
+
+extension B2BAuthHomeViewController: B2BEmailMagicLinksViewControllerDelegate {
+    func emailMagicLinkSent() {
+        Task { @MainActor in
+            let emailConfirmationViewController = EmailConfirmationViewController(state: .init(configuration: viewModel.state.configuration))
+            navigationController?.pushViewController(emailConfirmationViewController, animated: true)
+        }
+    }
+
+    func usePasswordInstead() {}
+}
+
+extension B2BAuthHomeViewController: B2BPasswordsHomeViewControllerDelegate {}
+
+extension B2BAuthHomeViewController: B2BSSOViewControllerDelegate {}
```

Comment:
Basically all the view controllers on the home screen that are child view controllers of home will relay events back so that navigation events can take place from here.